### PR TITLE
Update xcode plugin

### PR DESCRIPTION
**Pull request by eliottparis**
This PR update the plugin that brings syntax highlighting of the OTD language into Xcode.

This includes the following changes:

- The installer script has been replaced by a new one.
- Documentation has been added for plugin installation.
- More tokens are colorized (ie. text inside brackets).

### CODE REVIEW

File: scripts/xcode/otd.xclangspec [hunk 1]
```diff
@@ -29,6 +60,9 @@
         Syntax = {
             Start = "***";
             End = "***";
+            IncludeRules = (
+				"xcode.lang.url",
```
Comment (ohmtech-rdi) [inline scripts/xcode/otd.xclangspec]: Nit: this line and the next one are using tabs instead of space

File: scripts/xcode/otd.xclangspec [hunk 2]
```diff
@@ -37,66 +71,59 @@
...
-            Type = "xcode.syntax.plain";
-        };
-    },
-    {
-        Identifier = "xcode.lang.otd.lexer";
+        Name = "OTD";
         Syntax = {
             IncludeRules = (
                 "xcode.lang.comment",
+                "xcode.lang.url",
```
Comment (ohmtech-rdi) [inline scripts/xcode/otd.xclangspec]: 👍

**Review by ohmtech-rdi — APPROVED** [review]
👍

File: scripts/xcode/otd.xclangspec (hunk 1 above)
Comment (eliottparis) [reply scripts/xcode/otd.xclangspec]: Nice catch, I will also take the opportunity to format the whole file using _3 spaces indentation_ style.

**Pull request merged**